### ISSUE TITLE
vault_jwt_auth_backend doesn't support importing

### ISSUE THREAD

**Issue by rosteroster** [terraform-providers/terraform-provider-vault#295]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
v0.11.11

### Affected Resource(s)
Please list the resources as a list, for example:
- vault_jwt_auth_backend

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
resource "vault_jwt_auth_backend" "sl_iam_jwt_auth" {
  path               = "jwt"
  oidc_discovery_url = "https://iam.strln.net"
  bound_issuer       = "https://iam.strln.net"
}
```

### Expected Behavior
We should be able to do a terraform import for vault_jwt_auth_backend resources because otherwise we end up in a state where we can neither run "terraform apply" (it attempts and fails to create the backend again at the same path) nor can we import it to the terraform state easily, so an existing jwt backend puts us into a deadlock.

### Actual Behavior
terraform import fails saying that it's not supported by this resource type

Attempted to fix by adding 
```
Importer: &schema.ResourceImporter{
    State: schema.ImportStatePassthrough,
},
```
to the vault/resource_jwt_auth_backend.go schema.Resource but this started failing with the following error on import attempts:
```
vault_jwt_auth_backend.sl_iam_jwt_auth: Importing from ID "jwt"...
vault_jwt_auth_backend.sl_iam_jwt_auth: Import complete!
  Imported vault_jwt_auth_backend (ID: jwt)
vault_jwt_auth_backend.sl_iam_jwt_auth: Refreshing state... (ID: jwt)

Error: vault_jwt_auth_backend.sl_iam_jwt_auth (import id: jwt): 1 error(s) occurred:

* import vault_jwt_auth_backend.sl_iam_jwt_auth result: jwt: import vault_jwt_auth_backend.sl_iam_jwt_auth (id: jwt): Terraform detected a resource with this ID doesn't
exist. Please verify the ID is correct. You cannot import non-existent
resources using Terraform import.
```
We've confirmed via "vault auth list" & "vault auth read /auth/jwt/config" that this backend does in fact exist but the import is failing

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
The idea to that Importer: blah code segment came from the (functional) generic auth backend resource:
https://github.com/terraform-providers/terraform-provider-vault/blob/master/vault/resource_auth_backend.go#L20-L22

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by tyrannosaurus-becks** [comment]
FWIW, https://github.com/terraform-providers/terraform-provider-vault/pull/783 will add support for this by way of https://github.com/terraform-providers/terraform-provider-vault/pull/780. Please see 780 for docs on how to use it.

**Comment by catsby** [comment]
Hello there! This was patched in #806 and will go out in the next release (next few weeks). Thanks for your patience!
